admin_guide/iptables.adoc

```diff
@@ -11,18 +11,89 @@
 toc::[]
 
 == Overview
-This topic describes how administrators should work with iptables. openshift-sdn takes care of adding the
-necessary iptables rules to make it work. Kubernetes and Docker also manage iptables for port forwarding
-and services.
+There are many system components including {product-title}, Docker, and the Firewall that rely on
+the kernel iptables configuration for proper network operation. In addition the iptables
+configuration of all nodes in the cluster must be correct for networking to work.
 
+All components independently work with iptables without knowledge of how other components
+are using them. This makes it very easy for one component to break another component's configuration.
+Further, {product-title} and Docker assume that iptables remains set up exactly as they have set it up.
+They do not detect problems introduced by other components.
```

Comment:
s/problems/changes/

Also, OpenShift does monitor and fix, but there can be some lag.  Docker doesn't monitor.